3scale/porta
# THREESCALE-1699: Broken links in the admin portal

**What this PR does / why we need it**:

Updates broken links in the admin portal and rename "Follow the Quickstart guide" in Developer Portal > Content

**Which issue(s) this PR fixes** 

[THREESCALE-1699](https://issues.redhat.com/browse/THREESCALE-1699)

**Verification steps** 

1. Go to Products > Analytics > Response Codes and confirm link for "Response Codes Tracking" works
2. Go to Audience > Developer Portal > Content
3. Confirm that in the Quick Links section there's an item "Creating the Developer Portal" and that the link works.

**Response Codes**
<img width="1434" alt="Captura de pantalla 2023-07-20 a las 17 06 45" src="https://github.com/3scale/porta/assets/72191439/16bf822a-23c1-4cf0-a09b-e8eb3ea83d30">

**Quick Links**
<img width="1139" alt="Captura de pantalla 2023-07-20 a las 17 06 27" src="https://github.com/3scale/porta/assets/72191439/78ff7010-8771-4568-85e5-f88e76c8deca">
